YACReaderLibrary/server/controllers/v2/pagecontroller_v2.cpp

```diff
@@ -33,7 +31,6 @@ void PageControllerV2::service(HttpRequest &request, HttpResponse &response)
     bool remote = path.endsWith("remote");
 
     QStringList pathElements = path.split('/');
-    QString libraryName = DBHelper::getLibraryName(pathElements.at(2).toInt());
```

Comment:
This and the identical change in `PageController::service()` are the ones I'm not sure about. If the call to `DBHelper::getLibraryName()` is useful here, either the variable assignment should be removed or an explaining comment should be added.